src/pages/Accounts/index.tsx

```diff
@@ -230,6 +237,22 @@ export const AccountsPage = () => {
     setCurrentModal(undefined);
   }, []);
 
+  useEffect(() => {
```

Comment:
Extract it as a component please. So we can have easier time replacing it as well as less stuff/code on page.